test/donations.routes.test.js

```diff
@@ -29,7 +30,7 @@ const stripeMock = require('./mocks/stripe');
 /**
  * Tests.
  */
-describe('payments.routes.test.js', () => {
+describe('donations.routes.test.js', () => {
```

Comment:
I didn't update the old routes in tests intentionally since website was
still using them. Don't want to break them unintentionally.
On Tue, Apr 26, 2016 at 10:00 AM Sébastien Dubois <notifications@github.com>
wrote:

> In test/donations.routes.test.js
> <https://github.com/OpenCollective/api/pull/234#discussion_r61090100>:
>
> > @@ -29,7 +30,7 @@ const stripeMock = require('./mocks/stripe');
> >  /**
> >   * Tests.
> >   */
> > -describe('payments.routes.test.js', () => {
> > +describe('donations.routes.test.js', () => {
>
> Not sure why this test still calls .post('/groups/' + group.id +
> '/payments') (line 206) which is supposed to be a deprecated route cfr.
> comment
> https://github.com/OpenCollective/api/pull/234/files#diff-6db976b763ed71154781e45807c0d4e5R127
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenCollective/api/pull/234/files/29ec3f3523c35d7769909856960cb9da062cb8f0#r61090100>
>